### PR TITLE
[http] Add tracking for URL query string parameters

### PR DESCRIPTION
**Pull request by jd**
This adds a new field http.queryString that stores the query parameter passed
to the requested URL.

### CODE REVIEW

**Comment by jd** [comment]
This should be disabled by default I think. I have to check.

File: ddtrace/contrib/molten/patch.py [hunk 1]
```diff
@@ -123,6 +123,10 @@ def _w_start_response(wrapped, instance, args, kwargs):
 
         span.set_tag(http.METHOD, request.method)
         span.set_tag(http.URL, request.path)
+        span.set_tag(http.QUERY_STRING, '&'.join(
```
Comment (brettlangdon) [inline ddtrace/contrib/molten/patch.py]: `urlencode`?

File: ddtrace/contrib/flask/middleware.py [hunk 2]
```diff
@@ -158,6 +158,9 @@ def _finish_span(self, span, exception=None):
             method = request.method
             endpoint = request.endpoint or code
             url = request.base_url or ''
+            query_string = compat.to_unicode(request.query_string)
```
Comment (brettlangdon) [inline ddtrace/contrib/flask/middleware.py]: This Flask middleware is actually deprecated, nobody should be using this. You'll want to update `contrib/flask/path.py`.

https://github.com/DataDog/dd-trace-py/blob/4313f388383b90ccf2bcbca9d7ef1c400c827ece/ddtrace/contrib/flask/patch.py#L330

**Comment by jd** [comment]
@DataDog/apm-python Do we still want this? It's ready to go!

File: ddtrace/contrib/django/middleware.py [hunk 3]
```diff
@@ -138,6 +138,11 @@ def process_request(self, request):
             # Set HTTP Request tags
             span.set_tag(http.METHOD, request.method)
             span.set_tag(http.URL, get_request_uri(request))
+            trace_query_string = settings.TRACE_QUERY_STRING
+            if trace_query_string is None:
+                trace_query_string = config.django.trace_query_string
```
Comment (brettlangdon) [inline ddtrace/contrib/django/middleware.py]: 🙌

**Comment by brettlangdon** [comment]
seems fine given it is behind a config option and disabled by default, some people will want this.

**Pull request merged**